web/src/components/molecules/MyIntegration/WebhookForm/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Checkbox, { CheckboxOptionType } from "@reearth-cms/components/atoms/Checkbox";
+import Col from "@reearth-cms/components/atoms/Col";
+import Divider from "@reearth-cms/components/atoms/Divider";
+import Form from "@reearth-cms/components/atoms/Form";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import Row from "@reearth-cms/components/atoms/Row";
+import { useT } from "@reearth-cms/i18n";
+
+const WebhookForm: React.FC = () => {
+  const t = useT();
+  const [form] = Form.useForm();
+
+  const options: CheckboxOptionType[] = [
+    { label: t("Create"), value: "Create" },
+    { label: t("Update"), value: "Update" },
+    { label: t("Delete"), value: "Delete" },
+    { label: t("Accessed by API"), value: "Accessed by API" },
+  ];
+
+  const urlRegex =
```

Comment:
CMS has a regex file in utils, please move this there and import it in!
https://github.com/reearth/reearth-cms/blob/main/web/src/utils/regex.ts

Reply:
https://github.com/reearth/reearth-cms/blob/a282e0add4dc6a35a87b65d32871ff9f3fddcd4f/web/src/utils/regex.ts#L7-L8